{
	"118": {
		"0000": {
			"name": "nn::pia::ResultNexInternalError",
			"message": "",
			"short_description": "Unknown cause.",
			"long_description": "Unknown cause.",
			"short_solution": "Unknown solution.",
			"long_solution": "Unknown solution.",
			"support_link": "https://preten.do/118-0000"
		}
	}
}